src/reflow/mod.rs

```diff
@@ -27,6 +27,87 @@ impl Default for ReflowConfig {
 #[derive(Debug)]
 pub struct Reflow;
 
+impl Reflow {
+    /// Parse CommonMark files and reflow only neccessary portions of it
+    fn reflow_commonmark_chunks<'a, 's>(
+        origin: &ContentOrigin,
+        chunks: &'a Vec<CheckableChunk>,
+        config: &ReflowConfig,
+    ) -> Result<Vec<Suggestion<'s>>>
+    where
+        'a: 's,
+    {
+        let wrapper = textwrap::Wrapper::new(config.max_line_length)
+            .subsequent_indent("")
+            .initial_indent("");
+        unimplemented!();
+    }
+
+    /// Reflow chunks from rust source code
+    fn reflow_chunks<'a, 's>(
+        origin: &ContentOrigin,
+        chunks: &'a Vec<CheckableChunk>,
+        config: &ReflowConfig,
+    ) -> Result<Vec<Suggestion<'s>>>
+    where
+        'a: 's,
+    {
+        let wrapper = textwrap::Wrapper::new(config.max_line_length)
+            .subsequent_indent(" ")
+            .initial_indent(" ");
+        let suggestions = chunks
+            .iter()
+            .filter_map(|chunk| {
+                let mut new_lines = chunk
+                    .as_str()
+                    .split("\n\n")
+                    .collect::<Vec<&str>>()
+                    .iter()
+                    // @todo in doc comments, replacing newlines leads to a missing whitespace between words of
+                    // former last and first word. In comments, we always have the leading whitespace.
+                    // If we replace \n with ' ', this results in two whitespaces in normal comments
+                    // such that we have to replace them with a single whitespace
+                    .map(|s| s.replace("\n", " ").replace("  ", " ")) // @todo won't work for common mark
+                    .fold::<Vec<String>, _>(Vec::new(), |mut acc, comment| {
+                        let mut new_comment = wrapper
+                            .wrap_iter(comment.trim())
+                            .map(|b| b.into_owned())
```

Comment:
```suggestion
                            .map(into_owned)
```

Reply:
Do I need to include something with a `use` here? Your suggestion won't compile.

Reply:
Mean culpa, I think it must be `TheTraitIdent::into_owned`